Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/SplitBase.cs

```diff
@@ -0,0 +1,220 @@
+
+        /// <summary>
+        /// Distance (horizontal or vertical) to move, in response to keyboard activity.
+        /// </summary>
+        protected const double GripperKeyboardChange = 8.0d;
+
+        /// <summary>
+        /// Font family used for gripper.
+        /// </summary>
+        protected const string GripperDisplayFont = "Segoe MDL2 Assets";
+
+        /// <summary>
+        /// Gets or sets the content of the splitter control.
+        /// </summary>
+        public object Content
+        {
+            get { return (object)GetValue(ContentProperty); }
+            set { SetValue(ContentProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Content"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty ContentProperty =
+            DependencyProperty.Register(nameof(Content), typeof(object), typeof(SplitBase), new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets the content template for the <see cref="Content"/>.
+        /// </summary>
+        public DataTemplate ContentTemplate
+        {
+            get { return (DataTemplate)GetValue(ContentTemplateProperty); }
+            set { SetValue(ContentTemplateProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="ContentTemplate"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty ContentTemplateProperty =
+            DependencyProperty.Register(nameof(ContentTemplate), typeof(DataTemplate), typeof(SplitBase), new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets the cursor to use when hovering over the sizer.
+        /// </summary>
+        public CoreCursorType GripperCursor
+        {
+            get { return (CoreCursorType)GetValue(GripperCursorProperty); }
+            set { SetValue(GripperCursorProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="GripperCursor"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty GripperCursorProperty =
+            DependencyProperty.Register(nameof(GripperCursor), typeof(CoreCursorType), typeof(SplitBase), new PropertyMetadata(CoreCursorType.SizeWestEast));
+
+        /// <summary>
+        /// Gets or sets the foreground color of sizer grip.
+        /// </summary>
+        public Brush GripperForeground
+        {
+            get { return (Brush)GetValue(GripperForegroundProperty); }
+            set { SetValue(GripperForegroundProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="GripperForeground"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty GripperForegroundProperty =
+            DependencyProperty.Register(nameof(GripperForeground), typeof(Brush), typeof(SplitBase), new PropertyMetadata(default(Brush)));
```

Comment:
Let's clean this up and use a resource key in the XAML vs. a property.

Reply:
POTENTIAL BREAKING CHANGE: after inspecting if we even needed this `GripperForeground` property in the first place, I've been working on a further commit which would eliminate `GripperForeground` altogether, and devs can just use the built-in `Foreground` property to change the color, since the default Content of a `GridSplitter` is just a `TextBlock`. 

- Difficult to further explain here, as I haven't committed this change just yet, but you can compare and contrast `ContentSizer.xaml` (which does have a `ContentTemplate` defined) to `GridSplitter.xaml` (which doesn't _yet_ have a ContentTemplate, but will shortly, with some changes I'm cooking up)